internal/forms.py

```diff
@@ -35,6 +37,11 @@ class Meta:
 
     def __init__(self, user, **kwargs):
         super().__init__(**kwargs)
+
+        member = kwargs['instance']
+        if hasattr(member.user, 'card'):
+            self.initial['card_number'] = member.user.card.number.split()[1]  # Without EM prefix
```

Comment:
Here we again have the extra problems that prepending "EM" to the card numbers give.